x/stake/handler.go

```diff
@@ -163,7 +163,7 @@ func handleMsgEditValidator(ctx sdk.Context, msg types.MsgEditValidator, k keepe
 			return err.Result()
 		}
 		validator.Commission = commission
-		k.OnValidatorModified(ctx, msg.ValidatorAddr)
+		k.PostValidatorModified(ctx, msg.ValidatorAddr)
```

Comment:
This isn't post-modification? Changes haven't been written to the store yet (line 169).